content/tutorials/content-types/upload-content-with-a-custom-type.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Upload content with a custom type"
+metaTitle: "sensenet Tutorials - Upload content with a custom type"
+metaDescription: "sensenet Tutorials - Upload content with a custom type"
+---
+
+In this tutorial you will learn how you can upload content to a container with a custom type. We will look at how to upload other built-in types like Image or totally custom types that are inherited from the default File.
+
+## Upload basics
+
+By default if you upload a file into a container in sensenet it will be created as a `File`. `File` is the built-in type for storing files in the content repository. This types key field is a binary field, which contains the actual binary of the file. Learn more about the `File` content type in the [concepts](content/concepts/content-types/06-file).
```

Comment:
```suggestion
By default if you upload a file into a container in sensenet it will be created as a `File`. `File` is the built-in type for storing files in the content repository. This type's key field is a binary field, which contains the actual binary of the file. Learn more about the `File` content type in the [concepts](content/concepts/content-types/06-file).
```